gcovr/utils.py

```diff
@@ -280,13 +280,15 @@ def match(self, path):
 
 class DirectoryPrefixFilter(Filter):
     def __init__(self, directory):
-        abspath = os.path.realpath(directory)
+        abspath = realpath(directory)
+        if not abspath.endswith(os.sep):
+            abspath += os.sep
         os_independent_dir = abspath.replace(os.path.sep, '/')
-        pattern = re.escape(os_independent_dir + '/')
+        pattern = re.escape(os_independent_dir)
         super(DirectoryPrefixFilter, self).__init__(pattern)
 
     def match(self, path):
-        normpath = os.path.normpath(path)
+        normpath = os.path.normpath(path).replace(os.sep, "/")
```

Comment:
If we're already working on this, shouldn't the `normpath()` call be replaced by `realpath()`?

Reply:
I can check what happens. In my opinion normpath should be removed in all places.

Reply:
I've replaced this.